csrc/mmdeploy/net/rknn/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+project(mmdeploy_rknn_net)
+
+add_library(rknn SHARED IMPORTED)
+
+if( CMAKE_TOOLCHAIN_FILE )
```

Comment:
No need to include(CMAKE_TOOLCHAIN_FILE). 
toolchain file is used to specify the cross-compiler and is read early in the CMake run.
The `CMAKE_SYSTEM_PROCESSOR` or `CMAKE_SYSTEM_NAME` can be retrieved directly without including this file

Reply:
If don't include this, the cmake file could not be found. It is because I gave wrong `CMAKE_SYSTEM_PROCESSOR ` or `CMAKE_SYSTEM_NAME`?